resources/views/home.blade.php

```diff
@@ -1,23 +1,57 @@
 @extends('layouts.app')
 
 @section('content')
-<div class="container">
-    <div class="row justify-content-center">
-        <div class="col-md-8">
-            <div class="card">
-                <div class="card-header">Dashboard</div>
-
-                <div class="card-body">
-                    @if (session('status'))
-                        <div class="alert alert-success" role="alert">
-                            {{ session('status') }}
+    {{-- Header --}}
+    <header>
+        <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
+            <div class="mx-auto order-0">
+                <p class="header__title">Event Manager UFFS</p>
+            </div>
+
+            <div class="order-3">
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#headerUserSection" aria-controls="headerUserSection" aria-expanded="false">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+
+                <div class="collapse navbar-collapse" id="headerUserSection">
```

Comment:
Percebi que faltou a gente adicionar o padrão de nomes para ids. Achei um artigo e uma pergunta no StackOverflow que pode ajudar a gente a decidir isso.
 -  [What is the standard naming convention for html/css ids and classes?](https://stackoverflow.com/questions/6028211/what-is-the-standard-naming-convention-for-html-css-ids-and-classes)
 - [Naming Conventions in HTML](https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/naming-conventions-html.html)

Reply:
acho massa usar a convenção que separa por hífen, para não gerar confusão com nomes grandes: `long-word-id-name`, `longwordidname`